### PR TITLE
new signature recipes

### PR DESCRIPTION
**Pull request by Aniketjain1726**
updated signature recipes folder

### CODE REVIEW

File: recipes/signature_latte.txt
Comment (manyamittal09) [inline recipes/signature_latte.txt:1]: everything is good but kindly add the menu of hot chocolate

**Review by manyamittal09 — COMMENTED** [review]
everything is good but kindly add the menu of hot chocolate

**Comment by Aniketjain1726** [comment]
hot chococlate updated in the menu

**Comment by manyamittal09** [comment]
great idea of adding the hot chocolate in the menu.

**Review by manyamittal09 — APPROVED** [review]

**Pull request merged**